app/helpers/application_helper.rb

```diff
@@ -1,16 +1,22 @@
 module ApplicationHelper
   DEFAULT_GRAVATAR_SIZE = 60
   DEFAULT_SCREENHERO_URL = "http://screenhero.com"
+  GRAVATAR_FALLBACKS = {
+    wa: 'gravatar_fallback_wa.png',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.